src/resources/postcss/admin/category-colors/admin-category.pcss

```diff
@@ -0,0 +1,194 @@
+.table-view-list.tags {
+	.fixed .column-category_priority {
+		width: 75px;
+	}
+
+	.column-category_color {
+		width: 25%;
+
+		.tec-events-taxonomy-table__category-color-preview {
+			aspect-ratio: 1/1;
+			border-radius: 6px;
```

Comment:
There's a prop for this (--tec-border-radius-default: 4px) tweak if it works, else set up a new prop if we're going to use it again. If we're not, add a comment.